packages/flame/lib/src/fps_counter.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/scheduler.dart';
 
-import 'game/game.dart';
+import '../game.dart';
```

Comment:
would this file fit well inside the `game` directory?

Reply:
Indeed it would, in `game/mixins`.